manager/integration/tests/test_zone.py

```diff
@@ -0,0 +1,215 @@
+import pytest
+import time
+
+from common import client # NOQA
+from common import core_api  # NOQA
+from common import create_and_check_volume
+from common import get_self_host_id
+from common import RETRY_COUNTS
+from common import RETRY_INTERVAL
+from common import volume_name # NOQA
+from common import wait_for_volume_degraded
+from common import wait_for_volume_healthy
+from common import wait_for_volume_replica_count
+from random import randrange
+from test_scheduling import wait_new_replica_ready
+from common import get_version_api_client
+
+
+ZONE1 = "lh-zone1"
+ZONE2 = "lh-zone2"
+
+
+def get_k8s_zone_label():
+    ver_api = get_version_api_client()
+    k8s_ver_data = ver_api.get_code()
+
+    k8s_ver_major = k8s_ver_data.major
+    assert k8s_ver_major == '1'
+
+    k8s_ver_minor = k8s_ver_data.minor
+
+    if int(k8s_ver_minor) >= 17:
+        k8s_zone_label = "topology.kubernetes.io/zone"
```

Comment:
It's better to continue using constants for the label keys.

Reply:
fixed